constants.py

```diff
@@ -0,0 +1,18 @@
+class MappingIndex:
+    DATE = 0
+    MAXIMUM_TEMPERATURE = 1
+    MINIMUM_TEMPERATURE = 3
+    HUMIDITY = 7
+    SLICED_DATE = 2
+    AVERAGE_TEMPERATURE = 2
+    AVERAGE_HUMIDITY = 8
+    WEATHER_DIR = "weatherfiles"
+    RED_COLOR_CODE = '\033[91m'
+    BLUE_COLOR_CODE = '\033[34m'
+    RESET_COLOR_CODE = '\033[0m'
```

Comment:
create separate variables for these variables which should not be part of MappingIndex